cloudpeers/tlfs
# [WIP] wasm-bindgen & ffi-gen

changes to ffi-gen @ https://github.com/cloudpeers/ffi-gen/pull/60

to test
```
cd api
./build.sh
npx --yes http-server .
```
open browser and check console output:
```
RuntimeError: null function or function signature mismatch
    at <core::pin::Pin<P> as core::future::future::Future>::poll::hee1f14e768aa370f (05653dba:0xc3577a)
    at tlfs::api::FfiFuture<T>::poll::h76c2bf01723a3ed7 (05653dba:0x96beba)
    at tlfs::api::__create_memory_future_poll::{{closure}}::h662b2f4756108fd2 (05653dba:0x461870)
    at std::panicking::try::do_call::h2d6078dda0039457 (05653dba:0x87c3fc)
    at std::panicking::try::h92d29eb55d90adbf (05653dba:0x73cc07)
    at std::panic::catch_unwind::h3ccbc55a52d5dafb (05653dba:0xd1d2f9)
    at tlfs::api::panic_abort::hdc1653a7cab383da (05653dba:0x9225a4)
    at __create_memory_future_poll (05653dba:0xb40cef)
    at Api.createMemoryFuturePoll (bindings.mjs:655)
    at bindings.mjs:284
```

----

I worked around having to merge the `importObject` by providing the notifier callback via the global `window` object.
